src/main/java/core/basesyntax/service/impl/ReportMakerServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReportMakerService;
+import java.util.Map;
+
+public class ReportMakerServiceImpl implements ReportMakerService {
+    private static final String TABLE_HEADER = "fruit,quantity";
+    private static final char TABLE_SEPARATOR = ',';
+
+    @Override
+    public String generateReport(Map<String, Integer> data) {
+        StringBuilder report = new StringBuilder();
+        report.append(TABLE_HEADER).append(System.lineSeparator());
+        for (Map.Entry<String, Integer> pair : data.entrySet()) {
+            report.append(pair.getKey())
+                .append(TABLE_SEPARATOR)
+                .append(pair.getValue())
+                    .append(System.lineSeparator());
```

Comment:
```suggestion
            report.append(System.lineSeparator())
            .append(pair.getKey())
                .append(TABLE_SEPARATOR)
                .append(pair.getValue());
```

Reply:
otherwise, maven complains

Reply:
No way! My personal code is working. I don't know why maven will complain.